dev/Notifications/BaseNotifications/BaseNotification.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <wil/resource.h>
+#include <winrt/Windows.Foundation.Collections.h>
+#include <winrt/Microsoft.Windows.AppNotifications.h>
+#include <frameworkUdk/ToastNotificationsRT.h>
+
+namespace Microsoft::Windows::BaseNotifications {
+    struct BaseNotification
+    {
+        BaseNotification() = default;
+
+        BaseNotification(winrt::hstring const& payload);
+        winrt::hstring Tag() const noexcept;
+        void Tag(winrt::hstring const& value);
+        winrt::hstring Group() const noexcept;
+        void Group(winrt::hstring const& value);
+        uint32_t Id() const noexcept;
+        void Id(uint32_t id);
+        winrt::hstring Payload() const noexcept;
+        winrt::Microsoft::Windows::AppNotifications::AppNotificationProgressData Progress() const noexcept;
```

Comment:
These properties are not relevant for raw or badge, should not be part of base notifications,
similar comment for other properties in this class.

Reply:
We pass notification detail to FrameworkUdk as NotificationProperties. So NotificationProperties act as a adapter between WindowsAppSdk and FrameworUdk. And the INotificationProperties1 contains the Progress data so to use the existing functionality we should keep the progess data in the base class. Every other properties need to be part of base notifications because in WIndowsSdk also these are part of the base notifications

Reply:
I think, this can be managed through the specific notification classes like "ToastNotification" , "BadgeNotification", "RawNotification" ... The new Base Notification class need not be copy of AppNotifications class that we have currently, if we are modularizing this, specific properties should go to the relevant classes.

Reply:
Yes i agree but the way INotificationProperties was constructed these are all the basic properties that needs to be here. Let's discuss this offline

Reply:
changed done, Progress and conference config removed from base class